addons/godot-xr-tools/objects/grab_points/grab_driver.gd

```diff
@@ -30,6 +30,8 @@ var lerp_duration : float = 1.0
 ## Lerp time
 var lerp_time : float = 0.0
 
+## Player body
```

Comment:
Note to self, either `RemoteTransform3D` is not updating in time, or this is not our fix and doubling up on the move signals from our Player Body is overkill. Might remove changes to this file.

Reply:
Removed the code that is overkill.